File: uniAns.java
import java.util.Scanner;

public class uniAns{
	public static void main(String[] args) {
		int n;
		Scanner scan = new Scanner(System.in);

		while(true){
			n = scan.nextInt();
			if(n<=-100 || n>=100){
				System.out.println("Input must be integers of one or two digits");
			}else if(n!=42){
				System.out.println(n);
			}else if(n==42){
				break;
			}
		}
	}
}

File: Easy/uniAns/uniAns.c
#include <stdio.h>

int main(void){
	int n;

	while(1){
		scanf("%d",&n);
		if(n<=-100 || n>=100){
			printf("Input must be integers of one or two digits\n");
		}else if(n!=42){
			printf("%d\n",n);
		}else if(n==42){
			break;
		}
	}
	
	return 0;
}